@here/harp-mapview/lib/text/TextElementsRenderer.ts

```diff
@@ -646,9 +661,9 @@ export class TextElementsRenderer {
             gpuSize: 0
         };
 
-        for (const renderer of this.m_textRenderers) {
-            renderer.textCanvas.getMemoryUsage(memoryUsage);
-            renderer.poiRenderer.getMemoryUsage(memoryUsage);
+        for (const screenSpaceRenderer of this.m_screenSpaceRenderers) {
+            screenSpaceRenderer.textCanvas?.getMemoryUsage(memoryUsage);
+            screenSpaceRenderer.poiRenderer?.getMemoryUsage(memoryUsage);
```

Comment:
Now we're calling the same poiRenderer multiple times and adding up multiple times the same memory usage.

Reply:
resolved